ironfish-cli/package.json

```diff
@@ -150,4 +148,4 @@
     "url": "https://github.com/iron-fish/ironfish/issues"
   },
   "homepage": "https://ironfish.network"
-}
+}
```

Comment:
nit: revert this & check your editor settings so it's not modifying and removing the end of file newline like all of our other files have.

Reply:
done